i18n/en_GB/mail/membership_application_confirmation_sustaining/intro_start.twig

```diff
@@ -1,4 +1,3 @@
-Fantastic – you want to become a member of Wikimedia Deutschland e. V.
-We are pleased that you have decided to provide lasting support to the dissemination of free knowledge as a {$ membership_type $}.
+I'm very pleased with your decision to become a {$ membership_type $} of Wikimedia Deutschland e.V. – and thus part of a growing community of great people who are committed to effectively and sustainably supporting Wikipedia and promoting free knowledge. Thank you very much!
 
-What happens next? First, you will receive our welcome package {$ incentive $}by post to let you know that your membership has officially started. The welcome letter also gives you a transparent overview of all the information you need about your membership. However, since we send the welcome letter in bulk, please bear with us and give us a bit of time.
+What happens next? You will soon receive our welcome package {$ incentive $}by post – this also confirms that your membership has officially begun. However, since we always send the welcome packages together, we ask for your patience.
```

Comment:
```suggestion
What happens next? You will soon receive our welcome package {$ incentive $}by post – this also confirms that your membership has officially begun. However, since we always send the welcome packages in groups, we ask for your patience.
```